PageSpeed/GTmetrix/Common/ServicesManager.swift

```diff
@@ -8,7 +8,6 @@
 
 import Foundation
 
-// класс для хранения экземпляров сервисов, на время выполнения
 class ServicesManager {
```

Comment:
@ilyageekhub again, do we need this class? As we agreed it should be removed.

Reply:
This manager is necessary for keeping references of current active services. So its mandatory to keep it